back/.rspec

```diff
@@ -1,4 +1,2 @@
 --color
---require spec_helper
```

Comment:
Is it required anyway?

Reply:
`require 'spec_helper'` is part of `back/spec/rails_helper.rb` and `require 'rails_helper'` is at the top of all the spec files.